### PR TITLE
ENG-1984 / ENG-1986 - Add React Guides

### PR DESCRIPTION
**Pull request by djejaquino**
## Description

1. Adds Collect Cards with React Guide
2. Adds Collect PII with React Guide
3. Fixes consistency problems across Elements guides
4. Fixes ordering issue across all guides
5. Fixes code font

## Testing required outside of automated testing?

- [x] Not Applicable

#### Credit Card with React
localhost_4001_guides_collect-atomic-cards-with-elements-react_ (1)

#### PII with React
localhost_4001_guides_collect-pii-react_

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: collections/guides/collect_pii_react/index.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    - Pull background checks;
+- Make a third party API request with the raw data through our [Proxy](https://docs.basistheory.com/api-reference/#proxy).
+
+## See it in action
+
+In this example we also:
+- Pass a `mask` to the `ssn` field, restricting user input to digits only and formatting the output;
+- Use a `transform` option in the `ssn` field to strip out dash '-' characters from the tokenized data;
+- Disable the "Submit" `<button>` while the `fullName` field is empty or the `ssn` field is not `complete`, by listening to their `onChange` event;
+- Disable the both inputs and "Submit" `button` during the tokenization request inside the `submit` function;
```
Comment (lcschy) [inline collections/guides/collect_pii_react/index.md:115]: Disable both inputs and the "Submit" button

File: collections/guides/collect_cards_react/index.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+The created token in `submit` function can be used in a variety of ways:
+- You can fetch the card data later in your server, using a Server-to-Server [Application](https://docs.basistheory.com/api-reference/#applications) Key to call [Get an Atomic Card API](https://docs.basistheory.com/api-reference/#atomic-cards-get-an-atomic-card);
+- Call one of our [Reactors](https://docs.basistheory.com/api-reference/#reactors) to process that card data. [Here is how to charge a card with Node.js](/guides/collect-atomic-cards-with-elements/#setup-and-use-a-token-reactor);
+- Make a third party API request with the raw data through our [Proxy](https://docs.basistheory.com/api-reference/#proxy). Feel free to [contact us](mailto:support@basistheory.com?subject=CardElement%20and%20Proxy%20usage) for more information on how to do it.
+
+## See it in action
+
+In this example we also:
+- Disable the "Submit" `<button>` while the `CardElement` information is not `complete`, by listening to the `onChange` event;
+- Disable the `CardElement` and "Submit" `button` during the tokenization request inside the `submit` function;
```
Comment (lcschy) [inline collections/guides/collect_cards_react/index.md:104]: and **the** "Submit" button

File: collections/guides/collect_cards_react/index.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+          Submit
+        </button>
+    </>
+  );
+};
+```
+
+## Next steps
+
+The created token in `submit` function can be used in a variety of ways:
```
Comment (lcschy) [inline collections/guides/collect_cards_react/index.md:95]: in **the** submit

**Review by lcschy — CHANGES_REQUESTED** [review]
Just a couple of grammar comments. Content looks good.

File: collections/guides/collect_pii_react/index.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+          type="button"
+          onClick={submit}
+        >
+          Submit
+        </button>
+    </>
+  );
+};
+```
+
```
Comment (armsteadj1) [inline collections/guides/collect_pii_react/index.md:95]:
```suggestion
Success! 🎉, your application is now collecting and storing credit cards in Basis Theory platform. The example above uses a few of the [Element's user experience](https://docs.basistheory.com/elements/#introduction) features to ensure the best experience to your customers:

- Pass a `mask` to the `ssn` field, restricting user input to digits only and formatting the output;
- Use a `transform` option in the `ssn` field to strip out dash '-' characters from the tokenized data;
- Disable the "Submit" `<button>` while the `fullName` field is empty or the `ssn` field is not `complete`, by listening to their `onChange` event;
- Disable the both inputs and the "Submit" `button` during the tokenization request inside the `submit` function;
- Display the full `tokens` response;
- Log any tokenization errors to the `console`;

```

File: collections/guides/collect_pii_react/index.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+## See it in action
+
+In this example we also:
+- Pass a `mask` to the `ssn` field, restricting user input to digits only and formatting the output;
+- Use a `transform` option in the `ssn` field to strip out dash '-' characters from the tokenized data;
+- Disable the "Submit" `<button>` while the `fullName` field is empty or the `ssn` field is not `complete`, by listening to their `onChange` event;
+- Disable the both inputs and "Submit" `button` during the tokenization request inside the `submit` function;
+- Display the full stringified `tokens` response;
+- Log any tokenization errors to the `console`;
```
Comment (armsteadj1) [inline collections/guides/collect_pii_react/index.md:117]:
```suggestion

```
moved up

**Review by armsteadj1 — CHANGES_REQUESTED** [review]
Updated quite a few of the paragraphs and moved a few things around. Let me know if you think i moved too many thing. 🤣

File: collections/guides/collect_cards_react/index.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+    height: 630
+---
+# Collect Credit Cards with React
+
+Retain control of your card data while enabling your business to use any current or new processing relationships as you extend your business's product lines, look to increase auth rates, or reduce your overall cost of payments.  No matter why you're here, we will always enable you to use the data how you want to use it. Even if that means exporting the data out of Basis Theory.
+
+Basis Theory is here to make sure you can keep the lowest level of PCI compliance while retaining your ability to use your card data whenever and however you need to. We enable you to capture cards via form elements, exchange or forward that data to any other approved vendor or processor, and continue to grow and extend your business into its next growth cycle!
+
+If you'd like to follow along with this guide from scratch, we suggest creating a new React sandbox using <a href="http://codesandbox.io/">codesandbox.io</a> and getting started from there! Want to jump right into our sample app? <a href="https://codesandbox.io/s/github/Basis-Theory/basis-theory-js-examples/tree/master/collect-atomic-cards-with-elements-react">Find it here!</a>
```
[the remark this answers is not in the document]
Comment (djejaquino) [reply collections/guides/collect_cards_react/index.md:21]: Yes, but this is consistent across other examples. I would probably let this be reviewed/decided in that Guides task next cycle. What do you think?

File: collections/guides/collect_cards_react/index.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+- Call one of our [Reactors](https://docs.basistheory.com/api-reference/#reactors) to process that card data. [Here is how to charge a card with Node.js](/guides/collect-atomic-cards-with-elements/#setup-and-use-a-token-reactor);
+- Make a third party API request with the raw data through our [Proxy](https://docs.basistheory.com/api-reference/#proxy). Feel free to [contact us](mailto:support@basistheory.com?subject=CardElement%20and%20Proxy%20usage) for more information on how to do it.
+
+## See it in action
+
+In this example we also:
+- Disable the "Submit" `<button>` while the `CardElement` information is not `complete`, by listening to the `onChange` event;
+- Disable the `CardElement` and "Submit" `button` during the tokenization request inside the `submit` function;
+- Display the full stringified `token` response;
+- Log any tokenization errors to the `console`;
```
[the remark this answers is not in the document]
Comment (djejaquino) [reply collections/guides/collect_cards_react/index.md:106]: That's the thing. We don't. In the guide we do the very minimal to integrate. The live example that they can explore has those things. I am going to use your verbiage but stick with the same spot.

**Review by lcschy — APPROVED** [review]

File: collections/guides/collect_cards_react/index.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (armsteadj1) [reply collections/guides/collect_cards_react/index.md:19]: sounds good to me!

**Review by armsteadj1 — APPROVED** [review]

**Pull request merged**